org.ghidra_sre.Desktop.json

```diff
@@ -0,0 +1,64 @@
+{
+    "app-id" : "org.ghidra_sre.Desktop",
+    "branch": "stable",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "18.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.openjdk11"
+    ],
+    "command" : "ghidra",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--filesystem=home",
+         "--env=PATH=/usr/bin:/app/bin:/app/jdk/bin",
+         "--env=JAVA_HOME=/app/jdk"
+    ],
+    "modules" : [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": [
+              "/usr/lib/sdk/openjdk11/installjdk.sh"
+            ]
+        },
+        {
+            "name": "ghidra",
+            "buildsystem": "simple",
+            "build-commands": [
+                "for i in docs Extensions Ghidra GPL licenses server support ; do install -d -D $i /app/lib/ghidra/$i ; done",
+                "install -Dm755 ghidraRun /app/lib/ghidra/ghidraRun",
+                "install -Dm755 ghidra.sh /app/bin/ghidra",
+                "install -Dm644 docs/GhidraClass/AdvancedDevelopment/Images/GhidraLogo64.png /app/share/icons/hicolor/96x96/apps/org.ghidra_sre.Desktop.png",
```

Comment:
We need a 128 size for Flathub in case they have that.

Reply:
There isn't one readily available